configure.cmd

```diff
@@ -75,6 +75,21 @@ if not defined ARCH (
 
 set USE_CLANG_CL=0
 
+REM Some magic code to parse the parameters and extract those for CMake
+set CMAKE_PARAMS=
+set REMAINING=%*
+:loop
+for /f "tokens=1*" %%a in ("%REMAINING%") do (
+    set "PARAM=%%a"
+    if "!PARAM:~0,1!" == "-" (
+        if "!PARAM!" NEQ "-VS_VER" (
```

Comment:
Alternatively we could change the parameter to be `VS_VER=xx` without the dash to make it more consistent.